datadog_lambda/statsd_writer.py

```diff
@@ -1,16 +1,12 @@
 from datadog_lambda.stats_writer import StatsWriter
-from datadog import initialize, statsd
+from datadog_lambda.dogstatsd import statsd
```

Comment:
Can you add a comment in the other place (threadstats_writer?) where datadog is lazy loaded? just so people don't accidentally get import datadog into the critical path in the future?

Reply:
Good idea! I've updated both files with the comment. In threadstats_writer, there is no need to lazy load `datadog` as this file is lazy loaded itself here : https://github.com/DataDog/datadog-lambda-python/blob/main/datadog_lambda/metric.py#L30